install.sh

```diff
@@ -106,9 +118,18 @@ echo_standalone_postinstall() {
 		return
 	fi
 
+	channel=mainline
+	advisory="To install our stable release (v${STABLE_VERSION}), use the --stable flag. "
```

Comment:
nit: the trailing space might not be most convenient form, but I see why you have added this way, one condition less.